webapp/src/js/directives/content-row.js

```diff
@@ -66,10 +66,7 @@ angular.module('inboxDirectives').directive('mmContentRow', function() {
 
       // boolean: (optional) whether the contact is muted
       // html-minifier considers `muted` to be a boolean attribute, so it will strip its value
-      muted: '=isMuted',
-
-      // boolean: (optional) whether to display the "muted" text along with contact name
-      displayMuted: '='
```

Comment:
I'm pretty sure we still need displayMuted, right?

Reply:
Good catch .. fixed!